apps/frontend/src/main/mcp-server/start.sh

```
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# MCP Server Launcher with Electron Mocking
+
+# Mock Electron before starting
+export NODE_OPTIONS="--require $(dirname "$0")/mock-electron.js"
+
+# Run tsx with the MCP server
+cd "$(dirname "$0")/../../../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)